tools/protein_properties/protein_properties.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import sys
+from Bio import SeqIO
+from Bio.SeqUtils.ProtParam import ProteinAnalysis
+
+with open(sys.argv[2], 'w+') as result:
```

Comment:
Ok, this is a good argument. `sys.argv`will still be needed, or I also read from stdin.